### ISSUE TITLE
LU publishing failed when contains pattern with perbuilt entity 

### ISSUE THREAD

**Issue by KayMKM** [microsoft/BotFramework-Composer#2749]
## Describe the bug

LU publishing failed when contains patterns with perbuilt entity. E.g:
```
- (create|book|schedule) (a|an) (meeting|appointment|event) [with {personName}] (at|for|on) [{FromDate}] [{FromTime}]
@ prebuilt personName
```

The reason is the prebuilt entity will be removed when cross train.

## Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

**Comment by hibrenda** [comment]
Synced with Yue. This is the background.
- pattern with prebuilt entity can work for a single dialog 
- But it will hit some errors during #_interruption publish

**Comment by boydc2014** [comment]
One more simple repro case

- abc brenda
- abc {@personName}
@ prebuilt personName

**Comment by boydc2014** [comment]
we can use regexPattern="{.*?}" probably to detect an entity in pattern

**Comment by feich-ms** [comment]
BF-CLI fix is here https://github.com/microsoft/botframework-cli/pull/758. If we think this is P0 blocker of R9 release for composer, then BF-CLI should let above fix PR in before R9. Otherwise, we can target R10. Any thoughts @boydc2014

**Comment by boydc2014** [comment]
> BF-CLI fix is here [microsoft/botframework-cli#758](https://github.com/microsoft/botframework-cli/pull/758). If we think this is P0 blocker of R9 release for composer, then BF-CLI should let above fix PR in. Otherwise, we can target R10. Any thoughts @boydc2014 @hibrenda

I think we have chance to take that in CLI and cut a new RC.

**Comment by feich-ms** [comment]
BF-CLI fix is merged into 4.9 RC, please update the package in composer when next RC is online @lei9444
